src/NzbDrone.Common/Sonarr.Common.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="NLog" Version="4.7.14" />
     <PackageReference Include="NLog.Targets.Syslog" Version="6.0.3" />
     <PackageReference Include="NLog.Extensions.Logging" Version="1.7.4" />
-    <PackageReference Include="Sentry" Version="3.23.1" />
+    <PackageReference Include="Sentry" Version="4.0.0-beta.0" />
```

Comment:
We can merge this once `4.0.0` is out but hopefully we get reviews before that so the PR is ready to go.

Also happy to maintain the beta versions as we're close to releasing GA anyway